app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.test.tsx

```diff
@@ -27,7 +27,7 @@ describe("PageNew", () => {
   });
 
   describe("navigation links", () => {
-    const SidebarRoot = (props: RouteComponentProps) => (
+    const SidebarRoot: React.FunctionComponent<RouteComponentProps> = () => (
```

Comment:
Probably just a bikesheddy nit, but I haven't actually seen React.FunctionComponent type used before. Seems more verbose than what it's replacing and inconsistent with other functional components.

Guess it gets used when we don't actually use the `props` parameter directly? Seems like that's the case in this mock component and in most page components that use hooks for context.

Reply:
Yeah exactly, `props` isn't being used, so `no-unused-vars` hits it, but if you take it out entirely, typescript can't infer the type when `path` gets passed in at the call site, so typescript needs some way to know that that's a valid parameter, this is how you annotate the component with its props even if they aren't being used.